app/Models/Comment.php

```diff
@@ -124,6 +145,34 @@ public function getIsAutomatedAttribute(): bool
         return $this->user_id === self::SYSTEM_USER_ID;
     }
 
+    public function getUrlAttribute(): ?string
+    {
+        switch ($this->ArticleType) {
+            case ArticleType::Game:
+                return route('game.show', ['game' => $this->ArticleID, 'tab' => 'community']) . "#comment_{$this->ID}";
```

Comment:
If the comment is no longer in the recent comments, the link is ineffective and the result is just the game page.

Search on "primehack".

The first result just loads the game page without even jumping to the comments section.

http://localhost:64000/game/3154?tab=community#comment_3966491

The actual comment is on page 2:

http://localhost:64000/game/3154/comments?page=2#comment_3966491

Reply:
Merged latest `master` into this branch, which should address this feedback.